src/SkynetServer/Network/Client.PacketHandler.cs

```diff
@@ -113,8 +113,7 @@ public async Task Handle(P06CreateSession packet)
                     Session = await DatabaseHelper.AddSession(new Session
                     {
                         AccountId = confirmation.AccountId,
-                        ApplicationIdentifier = applicationIdentifier,
-                        CreationTime = DateTime.Now,
```

Comment:
If we no longer use `CreationTime`, then it should also be removed from the db model

Reply:
Creation times are generated by the database server. The value is necessary to create correct **`0x2C`**&nbsp;`DeviceListDetails` packets.